src/bun/lib/enonic/react4xp/asset/getImmutableDependencies.test.ts

```diff
@@ -0,0 +1,18 @@
+import {describe} from '@jest/globals';
+import {
+  expect,
+  it,
+} from 'bun:test';
+import {getImmutableDependencies} from '../../../../../main/resources/lib/enonic/react4xp/asset/getImmutableDependencies';
```

Comment:
Is it safe to import by a relative path? Looks a bit fragile.

Reply:
What do you mean fragile? :)

If you move either the imported module, or the test, a relative import must be updated.

I might be able to make bun work with tsconfig.json path.